terragrunt/aws/stepfunctions/iam.tf

```diff
@@ -0,0 +1,97 @@
+data "aws_iam_policy_document" "sfn_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["states.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "sfn_role" {
+  name               = "StepFunctionsRole"
```

Comment:
All of this IAM definition looks good, but could we use a more descriptive role name.  Maybe something like `GlueOrchestratorRole` to match the state machine name?

Reply:
Yup, totally fine with those names!